api/src/main/java/org/openmrs/api/db/PatientDAO.java

```diff
@@ -130,8 +133,17 @@ public interface PatientDAO {
 	 */
 	public List<Patient> getPatients(String name, List<PatientIdentifierType> identifierTypes,
 		boolean matchIdentifierExactly, Integer start, Integer length) throws DAOException;
-	
-	
+
+	/**
+	 * Get patient by universally unique identifier.
+	 * 
+	 * @param uuid universally unique identifier
+	 * @throws APIException
```

Comment:
Your throws annotation in the Javadoc should match the appropriate exception being thrown..."DAOException".